### DISCUSSION TITLE
Kaboom.rn

### DISCUSSION THREAD

**Discussion by AroliSG** [replit/kaboom#616] [Ideas]
Would be a great idea to move onto a new completely platform such as mobiles. With the tecnologic react native!

**Comment by lajbel** [comment]
Sounds a bit drastic. I think Kaboom is meant to be a more conventional engine, and for now it is moving forward on its way, soon there will be more support for mobiles!

**Comment by rawoke083** [comment]
oh please no !

If you want the pain (ok philosophy) of react there are a lot of other
libraries out there. I really enjoy the simplicity of kaboom and its
basically ecs(API at least) way of doing things.

What possible value could react bring to kaboom !?
